### ISSUE TITLE
Spacebar doesn't work in popout

### ISSUE THREAD

**Issue by jonwgeorge** [kakaroto/fvtt-module-popout#10]
When using popout, my players are unable to press the spacebar while typing. I've verified that it indeed doesn't work.

**Comment by kakaroto** [comment]
Confirmed the bug, though I'm baffled as to its cause.
It works in the editor, but just not in the input fields, though I don't see any reason for it. Affects both actor and journal sheets.

**Comment by jonwgeorge** [comment]
I did some digging today. This appears to only be happening on <input> fields that are rendered at the time of popout. I am able to edit things in the biography tab (of D&D 5E) just fine. I'm not sure if it's a system issue or a foundry issue just yet. Will keep digging.

**Comment by Dudrie** [comment]
I also looked into this with the following result: It seems that the popout window takes the command on pressing the "space" key to pause/unpause the game even if the field is focused (even though the actual game is _not_ actually getting paused). I have no idea why it does that (or how to properly fix it).

As one workaround I have replaced the `_onSpace` function in the KeyboardManager with a no-op. This works for now because pausing/unpausing from the popout did work anyway. But this is not a solid fix.

Alternativly the following code seems to work aswell (put beneath the overridden `_onEscape` function):
```js
const oldSpaceHandler = KeyboardManager.prototype._onSpace;
KeyboardManager.prototype._onSpace = function(event, up, modifiers) {
    oldSpaceHandler(this, event, up, modifiers);
}
```

**Comment by kakaroto** [comment]
Ha! I hadn't thought of space for pause/play. Makes sense. I'd have to dig through foundry's code to figure out why it thinks those fields should be handled that way.
Thanks for the analysis and workaround, I'll look into it!

**Comment by kakaroto** [comment]
this is fixed now the 0.8 release.
I didn't need to play around with the keyboard handler, I just had to mark the canvas as 'not ready' which made it fall through all the logic for the spacebar handling.
Thanks for the report and being patient for the fix!
